src/Controller/ThemeController.php

```diff
@@ -2,22 +2,48 @@
 
 namespace App\Controller;
 
+use App\Model\SubjectManager;
 use App\Model\ThemeManager;
 
 class ThemeController extends AbstractController
 {
+    private $themeManager;
     public const HEADERTITLE = 'KNOWLEDGE';
     /**
      * Display home page
      */
     public function index(): string
     {
-        $themeManager = new ThemeManager();
+
+        $this->themeManager = new ThemeManager();
```

Comment:
On veut faire ça dans le constructeur, comme ça, on a accès à $this->themeManager dans chaque fonction, et il sera déjà instancié

Reply:
Done